src/main.sh

```diff
@@ -714,13 +714,18 @@ install_package() {
 	for pkg in ${pkgname[@]}; do
 		fullver=$(NOEPOCH=1 get_full_version)
 		pkgarch=$(get_pkg_arch $pkg)
-		pkglist+=("${PKGDEST}/${pkg}_${fullver}_${pkgarch}.deb")
+		tmpfile=$(mktemp "/tmp/${pkg}_${fullver}_${pkgarch}_XXXXXX.deb")
```

Comment:
Can you wrap the `$()` call in quotes like this?

```sh
tmpfile="$(mktemp "/tmp/${pkg}_${fullver}_${pkgarch}_XXXXXX.deb")"
```

I can't think of why a call to `mktemp` would return anything with quoting issues, but I definitely prefer to keep all the code clean and careful when possible.